frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Meta, Story } from "@storybook/react";
+import GovInfoFormContainer from "./GovernmentInfoForm";
+
+export default {
+  component: GovInfoFormContainer,
+  title: "Government Info Form",
+} as Meta;
+
+const TemplateGovInfoForm: Story = () => {
+  return <GovInfoFormContainer />;
```

Comment:
The story should be of the internal component. We don't want components that interact with the api in storybook, they'll error on Chromatic